src/storyshots/commonConfig.ts

```
@@ -0,0 +1,17 @@
+import path from 'path'
+import { CommonConfig } from '@storybook/addon-storyshots-puppeteer'
+import { DirectNavigationOptions } from 'puppeteer'
+
+const storybookUrl = process.env.CI
+  ? `file://${path.resolve(__dirname, '../../storybook-static')}`
+  : 'http://localhost:6006'
+
+const commonConfig: Partial<CommonConfig> = {
+  storybookUrl,
+  getGotoOptions: (): DirectNavigationOptions => {
+    return process.env.CI ? { waitUntil: 'networkidle0' } : {}
+  }
```

Does including this option not break axe (just silently ignores I'm guessing?)

---

It's been a while since I committed this but I believe I was having issues with image snapshots when running against the local running version.